codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/protocols/rpcv2cbor/RpcV2CborCustomizations.kt

```diff
@@ -0,0 +1,15 @@
+package software.amazon.smithy.aws.swift.codegen.protocols.rpcv2cbor
+
+import software.amazon.smithy.aws.swift.codegen.AWSHTTPProtocolCustomizations
+import software.amazon.smithy.aws.swift.codegen.swiftmodules.AWSClientRuntimeTypes
+import software.amazon.smithy.codegen.core.Symbol
+import software.amazon.smithy.model.traits.TimestampFormatTrait
+
+class RpcV2CborCustomizations : AWSHTTPProtocolCustomizations() {
+
+    override val baseErrorSymbol: Symbol = AWSClientRuntimeTypes.RpcV2Cbor.RpcV2CborError
+
+    // Timestamp format is not used in RpcV2Cbor since it's a binary protocol. We seem to be missing an abstraction
+    // between text-based and binary-based protocols
```

Comment:
Based on SEP wording "`The [timestampFormat](https://smithy.io/2.0/spec/protocol-traits.html#timestampformat-trait) MUST NOT be respected to customize timestamp serialization.`", I assume this is the intended implementation

Reply:
will change or remove comment

Reply:
Nvm about my previous comment, on second look pretty sure the SEP is just saying disable timestamp format customization done via `@timestampFormat` trait.

Reply:
CBOR doesnt effect headers and timestampFormat trait is now bypassed when generating reading and writing closures: https://github.com/smithy-lang/smithy-swift/pull/887/files#diff-90d7c290bf90fafa9b697105ab1d98dda1e098df1a8db4d80c616efdb88df518R72

Reply:
Concluded offline that new commits to smithy-swift PR addresses the concern. Now timestamp members (regardless of whether it's top level or nested like in a list) will be serde'd using epoch seconds correctly without throwing an error